lib/fluent/plugin/out_file.rb

```diff
@@ -59,6 +60,16 @@ def configure(conf)
         conf['buffer_path'] ||= "#{@path}.*"
       end
 
+      test_path =
+        if @append
```

Comment:
Why this condition is needed?
This is for permission test. Adding `_0` causes different result?